warehouse/models/gtfs_views/_gtfs_views.yml

```diff
@@ -929,7 +929,7 @@ models:
          (not necessarily operating on date)
 
 exposures:
-  - name: calitp_reports_site
+  - name: legacy_calitp_reports_site
```

Comment:
I'd actually leave this for now and only rename once the reports site is migrated; see other comment on the new exposure (but a good idea to update this, just given the actual migration path I think it's a bit early to do this now)